src/ts/ContentMetadataBuilder.ts

```diff
@@ -91,8 +91,8 @@ export class ContentMetadataBuilder {
     convivaContentInfo[Conviva.Constants.DEFAULT_RESOURCE] = this.contentMetadata.defaultResource;
     convivaContentInfo[Conviva.Constants.STREAM_URL] = this.contentMetadata.streamUrl;
     convivaContentInfo[Conviva.Constants.IS_LIVE] = this.contentMetadata.streamType;
-    convivaContentInfo[Conviva.Constants.VIEWER_ID] = this.contentMetadata.viewerId;
-    convivaContentInfo[Conviva.Constants.PLAYER_NAME] = this.contentMetadata.applicationName;
+    convivaContentInfo[Conviva.Constants.VIEWER_ID] = this.contentMetadata.viewerId || 'GET_VIEWER_ID_FROM_PLAYER';
```

Comment:
If either of these are strings it gets rid of those integration errors. Basically they exist on the first session because they are called from the example app index.html file

    conviva.updateContentMetadata({
      applicationName: 'Bitmovin Player Conviva Analytics Integration Test Page',
      viewerId: 'uniqueViewerId',
      framework: 'Bitmovin Player',
      frameworkVersion: player.version,
      custom: {
        appVersion: '1.0',
        contentId: 'someContentId',
        playerVendor: 'bitmovin',
        playerVersion: player.version
      },
    });
    
  Once the user hits replay at the end of the stream all that data has been lost so bitmovin will have to find a way to update that. This bug existed in the previous version